cmd/ocm/list/region/cmd.go

```diff
@@ -71,18 +108,11 @@ func run(cmd *cobra.Command, argv []string) error {
 	// `Enabled` field checked only on Ret Hat infra, all regions supported with GCP CCS,
 	// account-specific list on AWS CCS.
 	// Use appropriate columns to make it easier to understand, with some always true.
-	if args.provider == "aws" && args.ccs.Enabled {
-		fmt.Fprintf(writer, "ID\t\tCCS (THIS AWS ACCOUNT)\tSUPPORTS MULTI-AZ\n")
-		for _, region := range regions {
-			fmt.Fprintf(writer, "%s\t\t%v\t%v\n",
-				region.ID(), true, region.SupportsMultiAZ())
-		}
-	} else {
-		fmt.Fprintf(writer, "ID\t\tON RED HAT INFRA\tON CCS\tSUPPORTS MULTI-AZ\n")
-		for _, region := range regions {
-			fmt.Fprintf(writer, "%s\t\t%v\t%v\t%v\n",
-				region.ID(), region.Enabled(), true, region.SupportsMultiAZ())
-		}
+
+	fmt.Fprintf(writer, "ID\t\tON RED HAT INFRA\tCCS ONLY\tSUPPORTS MULTI-AZ\n")
```

Comment:
I'm not sure about displaying the new columns here.
A "CCS ONLY column" is round-about: it's actually about RH infra, and it's negative (CCS ONLY = true means no RH infra, false = can use on RH infra).  And carries an unspoken implication that all regions work on CCS?

My goal in previous design of this table was to answer simple questions of what you can use in which scenarios, without forcing user to understand our exact rules.
So instead of explaining, I had a CCS column filled with "true"; I think giving a independent answer for the 2 scenarios is still a good goal.

Also, having an ON RED HAT INFRA column alongside CCS ONLY sounds confusing to me.  For example if both say "true", isn't that a contradiction?

I'm confused now about what `.enabled` field means (and perhaps I was always confused when I last changed this table?).

- The case when user passes --aws-... flags is more complex.
  We call the same `GetRegions()` function, but the meaning of the result changes, we get a subset relevant to user's AWS account.  I propose that modifying the table headers for that case is still useful.